classify_val_only.py

```diff
@@ -0,0 +1,16 @@
+import tlc
+
+from ultralytics.utils.tlc.detect.model import TLCYOLO
+from ultralytics.utils.tlc.detect.settings import Settings
+
+model = TLCYOLO("yolov8n-cls.pt")
+
+settings = Settings(
+    image_embeddings_dim=2,
+)
+
+# data + split should be okay
```

Comment:
cleanup

Reply:
Agreed to remove these files later. Leaving this up until they are deleted.